packages/desktop-client/src/components/App.tsx

```diff
@@ -38,37 +44,49 @@ type AppInnerProps = {
 function AppInner({ budgetId, cloudFileId }: AppInnerProps) {
   const [initializing, setInitializing] = useState(true);
   const { showBoundary: showErrorBoundary } = useErrorBoundary();
-  const stateLoadingText = useSelector((state: State) => state.app.loadingText);
-  const [loadingText = stateLoadingText, setLoadingText] = useState<
-    string | null
-  >(null);
-  const { loadBudget, closeBudget, loadGlobalPrefs } = useActions();
+  const loadingText = useSelector((state: State) => state.app.loadingText);
+  const dispatch = useDispatch();
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏ohhhhh... right, the local state I implemented does not listen to `state.app.loadingText` updates. 🤦 
Thanks for patching this!